frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -557,6 +565,32 @@ const applyFilters = <T, S>(
   )
 }
 
+function mergeProps(...props: any) {
+  const firstProps = props[0]
+  const restProps = props.slice(1)
+
+  if (!restProps.length) {
+    return mergeWith({}, firstProps)
+  }
+
+  return mergeWith(mergeWith({}, firstProps), ...restProps, (a: any, b: any, key: any) => {
+    if (key === 'children') {
+      if (a && b) {
+        return cloneElement(a, {
+          children: b,
+        })
+      }
+      return { ...b, ...a }
+    }
+
+    if (key === 'className') {
+      return css(a, b)
```

Comment:
In your code, this is `css` from `@emotion/css` and in the original code it is from `@patternfly/react-styles` which is a simpler function that just appends the class names. That would be sufficient for this case, rather than generating a new class name with emotion.